### PR TITLE
Set a separate memory limit for CLI PHP

### PR DESCRIPTION
**Pull request by byrond**
Fixes issue #34.

### CODE REVIEW

File: conf/vagrant/provisioning/roles/php/tasks/main.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   become: true
-  lineinfile: dest={{ item }}
+  lineinfile: dest={{ item.file }}
               regexp='memory_limit'
-              line='memory_limit = {{ php_ini_memory_limit }}'
+              line='memory_limit = {{ item.value }}'
   with_items:
-    - /etc/php/5.6/apache2/php.ini
-    - /etc/php/5.6/cli/php.ini
+    - ( file: '/etc/php/5.6/apache2/php.ini', value: "{{ php_ini_memory_limit }}" }
```
Comment (becw) [inline conf/vagrant/provisioning/roles/php/tasks/main.yml]: Is there a syntax error here? This line (and the next) start with a parenthesis `(` and end with a bracket `}`

**Review by becw — COMMENTED** [review]
I like this, generally. The other solution to this issue, though, is to run `composer self-update` on the VM, since composer has had some significant fixes in the past 6 months related to memory management.

Perhpas that should happen as part of the `Vagrantfile` (which would keep composer more up to date) or the provisioning (which would at least ensure that it's up to date at first).

File: conf/vagrant/provisioning/roles/php/tasks/main.yml (hunk 1 above)
Comment (byrond) [reply conf/vagrant/provisioning/roles/php/tasks/main.yml]: I'd say so.  Yikes!

**Comment by byrond** [comment]
I will try your suggestion and report back whether it looks like this change is necessary.  Thanks, @becw!

**Pull request merged**